webapp/src/Service/DOMJudgeService.php

```diff
@@ -143,6 +143,11 @@ public function getCurrentContests(?int $onlyofteam = null, bool $alsofuture = f
         return $qb->getQuery()->getResult();
     }
 
+    public function getCurrrentContestId(): ?int
```

Comment:
I would name this more specifically as to relating to the cookie. Now it "sounds" to me related to getCurrentContest() and getCurrentContests() while in fact it's slightly different.